<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiAddDiagnos</class>
 <widget class="QWidget" name="UiAddDiagnos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>461</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layGen">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 25 14pt &quot;Microsoft YaHei UI Light&quot;;</string>
      </property>
      <property name="text">
       <string>Nouveau diagnostique</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Nom du patient</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Date du diagnostique</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="date">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btnAddPatient">
          <property name="text">
           <string>Nouveau</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>img/Usual/New or add/add_file.png</normaloff>img/Usual/New or add/add_file.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbNom"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnRefreshCmb">
          <property name="text">
           <string>Rafraichir</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>img/Usual/Refresh/recurring_appointment.png</normaloff>img/Usual/Refresh/recurring_appointment.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Observation</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Correction</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTextEdit" name="txtObsrvation"/>
      </item>
      <item row="3" column="1">
       <widget class="QTextEdit" name="txtCorrection"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnValider">
        <property name="text">
         <string>Valider</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>img/Usual/Details/view_details.png</normaloff>img/Usual/Details/view_details.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAuit">
        <property name="text">
         <string>Annuler</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>img/Usual/Quit/close_window.png</normaloff>img/Usual/Quit/close_window.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnAuit</sender>
   <signal>clicked()</signal>
   <receiver>UiAddDiagnos</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
